src/oc/web/stores/reaction.cljs

```diff
@@ -156,9 +155,8 @@
   [db interaction-data add-event?]
   (let [item-uuid (:resource-uuid interaction-data)
         comments-key (get @reactions-atom (make-comment-index item-uuid))
-        all-posts-key (assoc comments-key 2 :all-posts)
-        ;; look for data in the board and in AP key
-        keys (remove nil? [comments-key all-posts-key])
+        ;; look for data in the board
+        keys (remove nil? [comments-key])
```

Comment:
remove nil? is not needed anymore since coments-key is the only key used and will always be not null